tests/phpunit/unit-tests/user/class-llms-test-user-permissions.php

```diff
@@ -164,6 +164,40 @@ public function test_user_can_manage_user_multiple_roles() {
 
 	}
 
+	/**
+	 * Test the editable_roles() filter for users with multiple roles.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_editable_roles_multiple_roles() {
+
+		$users = $this->create_mock_users();
+
+		$all_roles = LLMS_Roles::get_roles();
```

Comment:
The array of roles to feed the method `editable_roles` must be similar to the one passed by the `'editable_roles'` filter: https://developer.wordpress.org/reference/hooks/editable_roles/
so:

```suggestion
		$all_roles = wp_roles()->roles;
```